source/features/clean-dashboard.css

```diff
@@ -12,21 +12,30 @@
 	display: none;
 }
 
-/* Add background and border Whole list */
-.rgh-clean-dashboard .news > .f4 + div {
-	background-color: var(--rgh-background);
-	border: 1px solid var(--rgh-border-color);
-	border-radius: 6px;
-	margin-top: 8px !important;
-}
-
-/* Drop "box" style for all events */
-.rgh-clean-dashboard .news > .f4 + div .Box {
+/* Drop "box" style from the repos */
+.rgh-clean-dashboard .news .body .Box {
 	padding: 0 !important;
 	border: none;
 	background: none;
 }
 
+/* Add "box" style to each event group */
+/* This exact `>` sequence ensures it works on ajaxed-in events */
+.rgh-clean-dashboard [role='tabpanel'] [data-repository-hovercards-enabled] > [class] > .body,
+/* The organization newsfeed doesn't yet support the new style */
+.rgh-clean-dashboard .news > [data-repository-hovercards-enabled] > [class] > .body {
```

Comment:
This last selector doesn't apply to the new layout because `news` has more elements until `data-repository-hovercards-enabled`